components/sidebar/sidebar_category/sidebar_category.tsx

```diff
@@ -76,6 +78,9 @@ export default class SidebarCategory extends React.PureComponent<Props, State> {
             this.categoryTitleRef.current.addEventListener(A11yCustomEventTypes.ACTIVATE, this.handleA11yActivateEvent);
             this.categoryTitleRef.current.addEventListener(A11yCustomEventTypes.DEACTIVATE, this.handleA11yDeactivateEvent);
         }
+        if (!this.props.totalUserCount) {
+            this.props.actions.getTotalUsersStats();
```

Comment:
I hate to say it, but this isn't going to work here since we're going to be loading the total user count multiple times when the sidebar is mounted because each SidebarCategory is going to call this. I think the InviteMembersButton should be the one responsible for this.

To fix the test errors you were hitting before, couldn't you just mock out the action?

Reply:
shouldn't this ` if (!this.props.totalUserCount) {` make sure that it is called only once, and the next time it will grab the prop from the redux-store?